apps/website/src/components/Playground/index.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { Sandpack } from '@codesandbox/sandpack-react';
+import { useColorMode } from '@docusaurus/theme-common';
+import AppCode from '!!raw-loader!./template/App.js';
+import StylesCode from '!!raw-loader!./template/Styles.js';
+
+export default function Playground() {
+  const { isDarkTheme } = useColorMode();
+  const sandpackTheme = isDarkTheme ? 'sandpack-dark' : 'github-light';
```

Comment:
```suggestion
  const sandpackTheme = isDarkTheme ? 'dark' : 'github-light';
```

`dark` theme looks more matching to ours IMO.